packages/core/types/src/modules/documents/service-instance.ts

```diff
@@ -22,11 +29,7 @@ export type ServiceInstance<
   delete: <TParams extends Params.Delete<TContentTypeUID>>(
     documentId: ID,
     params?: TParams
-  ) => Result.Delete<TContentTypeUID, TParams>;
-
-  deleteMany: <TParams extends Params.DeleteMany<TContentTypeUID>>(
-    params: TParams
-  ) => Result.DeleteMany;
+  ) => Result.Delete;
```

Comment:
Are we missing deleteEntry from the document engine? And based on what it does should we rename that deleteComponent?

Reply:
We can come back to that later as we plan other refactorings :)